staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/helpers.go

```diff
@@ -228,6 +228,12 @@ func NewUIDPreconditions(uid string) *Preconditions {
 	return &Preconditions{UID: &u}
 }
 
+// NewPreconditionDeleteOptionsWithResourceVersion returns a DeleteOptions with a UID precondition set.
```

Comment:
Should say
`// NewPreconditionDeleteOptionsWithResourceVersion returns a DeleteOptions with a ResourceVersion precondition set.`